packages/form-system/stories/05-creating-a-custom-field.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import '@lion/input/lion-input.js';
+import { Validator } from '@lion/validate';
+import './helper-wc/h-output.js';
+
+<Meta title="Form Fundaments/Creating a Custom Field" />
+
+# Creating a custom field
+
+Custom fields can be created in just a few steps. All you need is an interaction element
+(like for instance a slider, a listbox or a combobox) and connect it to the [Field](../README.md)
+functionality.
+
+## Prerequisite: an interaction element
+
+An interaction element provides the means for the end user to enter a certain value, just like
+native elements provide in this (think of `<input>`, `<textarea>` and `<select>`).
```

Comment:
What does the phrase above try to convey? I'm a bit uncertain what we're trying to get across to people...

Reply:
ah, it was explained here (interaction element). 
You have to be a bit familiar with the FIeld and its subclasses like Input, Textarea etc.
Can you please elaborate on what is unclear and do a different proposal?

Again, changing this text was not the scope of this story ;)